aspnetcore/mvc/views/view-components.md

```diff
@@ -107,6 +107,12 @@ Pascal-cased class and method parameters for Tag Helpers are translated into the
 </vc:[view-component-name]>
 ```
 
+Note that in order to use a View Component as a Tag Helper, you must register the assembly containing the View Component using the `@addTagHelper` directive. For example, if your View Component is in an assembly called "MyWebApp" you will need the following directive in your `_ViewImports.cshtml` file to register the View Components:
```

Comment:
Yeah, I went back and forth about that. I had it phrased as "in your `_ViewImports.cshtml` file, or in any pages that use the View Component" but I kinda wanted to keep it simple. Might see if I can link to the existing Tag Helper registration docs: https://docs.microsoft.com/en-us/aspnet/core/mvc/views/tag-helpers/intro#managing-tag-helper-scope